Coding-Coach/coding-coach
# Adding auth0 to the clientside

## Description
This PR adds auth0 to the clientside. There's only one page protected, currently there are no links on the landing page, we don't want people to start registering yet because we don't have anything in the dashboard 😅

## Steps to QA

1. Open `/app/dashboard`
2. You should see an error message and a login button
3. Clicking the login button will redirect the user to auth0
4. After the user completes registration they will be able to see the dashboard
5. Users can logout, after that they won't have access to `/app/dashboard` anymore

Closes #223
